README.md

```diff
@@ -6,3 +6,5 @@ Xtext Gradle Plugins
 A set of [Gradle](http://gradle.org) plugins to build and use [Xtext](http://xtext.org) languages and the [Xtend](http://xtend-lang.org) programming language.
 
 The documentation is hosted on the [project's website](http://xtext.github.io/xtext-gradle-plugin/).
+
+To build and install locally run `./gradlew clean build install -Dorg.gradle.project.version=<version>` e.g. with `<version>` set to `1.0.22-SNAPSHOT`.
```

Comment:
There's no reason to clean and there's no reason to run the full build task if all you wanted was to install.

Simply run `./gradlew install`.

Reply:
bit this has no version